### PR TITLE
[res] Add quant test Quant_Conv_Mul_Add_001

### PR DESCRIPTION
**Pull request by jinevening**
This adds quant test Quant_Conv_Mul_Add_001.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/pull/8618

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_001/test.rule [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# To check mixed-precision quantization.
+# Conv, Mul: int16, Add: u8
+# Quantize Ops are inserted before Conv and after Mul.
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "CONV_INT16"              $(tensor_dtype ofm_conv) '=' INT16
+RULE    "WEIGHTS_INT16"           $(tensor_dtype filter) '=' INT16
+RULE    "BIAS_INT32"              $(tensor_dtype bias) '=' INT64
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_001/test.rule:9]: optional) `BIAS_INT32` has `INT32` but result is `INT64`. maybe match to `BIAS_INT64`?

**Review by seanshpark — DISMISSED** [review]
LGTM

File: res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_001/test.rule (hunk 1 above)
Comment (jinevening) [reply res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_001/test.rule]: Oops.. Thanks for finding!

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**